avocado/core/nrunner/runnable.py

```diff
@@ -140,16 +149,41 @@ def identifier(self):
 
         return fmt.format(**options)
 
+    @property
+    def config(self):
+        return self._config
+
+    @config.setter
+    def config(self, config):
+        """Sets the config values based on the runnable kind.
+
+        :param config: A config dict with new values for Runnable.
```

Comment:
It would nice to explain better this config here, making it clear this is not avocado config, but 'runnable config' only.

Reply:
Sure, I will do that.

Reply:
sure